src/main/java/arm32x/minecraft/commandblockide/client/CommandChainTracer.java

```diff
@@ -1,15 +1,15 @@
 package arm32x.minecraft.commandblockide.client;
 
-import java.util.*;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.Blocks;
 import net.minecraft.block.CommandBlock;
 import net.minecraft.client.world.ClientWorld;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.Direction;
 
+import java.util.*;
+import java.util.stream.Stream;
+
```

Comment:
These imports were reordered for no reason. In Command Block IDE, all imports are sorted alphabetically, including imports from the `java` package.

This was probably caused by your IDE configuration.